protocol/query/method/message/data/dataCloseRollCall.json

```diff
@@ -7,10 +7,15 @@
 	"properties": {
 		"object": { "const": "roll_call" },
 		"action": { "const": "close" },
-		"id": {
+		"update_id": {
 			"type": "string",
 			"contentEncoding": "base64",
-			"$comment": "Hash : SHA256('R'||lao_id||creation||name)"
+			"$comment": "Hash : SHA256('R'||lao_id||closes||end)"
+		},
+		"closes": {
+			"type": "string",
+			"contentEncoding": "base64",
+			"$comment": "The 'update_id' of the latest roll call open, or in its absence, the 'id' field of the roll call creation"
 		},
 		"start": { "description": "[Timestamp] start time", "type": "integer", "minimum": 0 },
```

Comment:
I agree with @gnarula, the `start` field represents the open/reopen time, which seems redundant because this information is present in the `closes` field.